modules/lsfilter/src/js/LSFilterInputWindow.js

```diff
@@ -19,7 +19,7 @@ $().ready(function() {
 			return false;
 		}
 	});
-	$(document).on("hover", "#filter_result table td a span.x16-add-comment", function() {
+	$(document).on("mouseenter", "#filter_result table td a span.x16-add-comment", function() {
```

Comment:
I don't understand the purpose of this event listener. But, I mean, sure, considering what it does I guess this is equivalent.